rfcs/proposed/simplified_numa_support/increased_availability/README.org

```diff
@@ -0,0 +1,101 @@
+# -*- fill-column: 80; -*-
+
+#+title: Improve predictability of API for NUMA support
+
+*Note:* This is a sub-RFC of the https://github.com/oneapi-src/oneTBB/pull/1535.
+Specifically, its section about "Increased availability of NUMA support".
+
+* Introduction
+oneTBB has soft dependency on several versions of ~tbbbind~, which are loaded by
```

Comment:
```suggestion
oneTBB has a soft dependency on several versions of ~tbbbind~, which are loaded by
```

Reply:
I additionally changed "versions" to "variants" to avoid confusion with DLL/SO versions.